test/parallel/test-buffer-fill.js

```diff
@@ -350,9 +350,8 @@ assert.strictEqual(
     };
     Buffer.alloc(1).fill(Buffer.alloc(1), 0, end);
   }, {
-    code: 'ERR_INDEX_OUT_OF_RANGE',
-    type: RangeError,
-    message: 'Index out of range'
```

Comment:
We might want to check at least a few messages. That way we make sure the manual part is generating a nice output.

Reply:
Sure! Done in all three places that I removed them.